hBank/src/main/java/salaryCalculator/SalaryCalculator.java

```diff
@@ -0,0 +1,68 @@
+package salaryCalculator;
+
+import java.text.DecimalFormat;
+
+import static salaryCalculator.TestNumbers.getInputTillIsANumber;
+
+public class SalaryCalculator extends CalculatorModel {
+    private final DecimalFormat decimalFormat = new DecimalFormat("0.00");
+
+    public void infoInput(){
+        System.out.println("Ievadīt Bruto algu");
+        double grossSalary = getInputTillIsANumber();
+        setGrossSalary(grossSalary);
+        //=====================================================
```

Comment:
Neskatoties uz to, ka liekas, ka šie varētu palīdzēt kārtot kodu, no tādiem vajadzētu izvairīties. Ilgtermiņā tikai sagādās problēmas. 
Ieteikums - ja metode ir tik liela, ka prasās to kaut kā sadalīt/atdalīt, tad tā ir laba zīme/ indikators, ka šo kodu var sadalīt mazākās metodes vai kā citādi uzlabot